docs/mosviz/navigation.rst

```diff
@@ -2,7 +2,17 @@
 Navigating the UI
 *****************
 
-Overview here...
+The Mosviz User Interface contains an image viewer in the top left section and
+a 2D spectrum viewer over a specviz viewer in the top right section. The bottom
+section houses the :ref:`table viewer<table-viewer>`.
+The image viewer contains all of the same functionality as an image viewer
+in Cubeviz, with the addition that there is a :ref:`slit overlay<slit-overlay>` that uses data from a
+spectrum 2d file that is linked to the image file to determine where to plot the slit.
+The 2D spectrum viewer and the specviz viewer have their x axes’ locked, meaning that
+when you Pan/Zoom in one viewer, the exact same action happens in the other.
```

Comment:
```suggestion
The 2D spectrum viewer and the 1D spectrum viewer viewer have their x axes locked to one another, meaning that
when you pan or zoom the x axis in one viewer, the same action happens in the other.
```

Minor wording suggestions

Reply:
Good suggestions! Yeah I had it as "1D spectrum viewer" originally but I wasn't sure if we wanted to stick with that terminology or start to shift towards "specviz viewer". I agree though that it's better to stay consistent until we all agree on something else.